src/sw.ts

```diff
@@ -538,6 +538,7 @@ async function fetchHandler ({ path, request, event }: FetchHandlerArg): Promise
       log.error('fetchHandler: response not ok: ', response)
       return await errorPageResponse(response)
     }
+    response.headers.set('ipfs-sw', 'true')
```

Comment:
playwright doesn't allow us to check if a response is from a service worker in firefox. This will likely help us moving forward and is the same namespace i've been using for assets as well